flask_common/mongo/__init__.py

```diff
@@ -1,4 +1,27 @@
 from .documents import DocumentBase, RandomPKDocument, SoftDeleteDocument
+from .fields import LowerEmailField, LowerStringField, TrimmedStringField
 from .querysets import ForbiddenQueryException, ForbiddenQueriesQuerySet
 from .query_counters import custom_query_counter
 from .utils import fetch_related, iter_no_cache
+
+# Import fields that exist only if some dependencies are installed.
+
+try:
+    from .fields import EncryptedStringField
```

Comment:
Since the idea of this PR is to isolate all mongo-specific stuff into a separate package, I don't see value of skipping those anymore.

If client code don't use mongo, they have no business even trying to import _anything_ from `flask_common.mongo`. And if they do use mongo and have issues with it (for example wrong version), the try-except would swallow the root cause of any actual issues they might have - but would fail _anyway_. All the client would see is `cannot import name EncryptedStringField` instead what's really wrong.

Reply:
I don't think these specific fields are skipped because of MongoEngine. In this particular case, I believe the field is skipped so that we can use `flask_common` w/o installing a heavy `pycrypto` package.